apps/docs/content/features/pricing.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Pricing
+description: Detailed pricing description for deployment on zerops.
+---
+
+import PricingCalculator from "@site/src/components/PricingCalculator"
+
+Zerops offers a usage based pricing model which can accommodate any **small-scale** or **high-availability** applications.
+
+## Projects
+
+A [project](/features/infrastructure#project) is the internal **infrastructure**, dedicated for your **services**. Think of it as a private network where you can deploy your services inside and interact with each other.
```

Comment:
I am not entirely sure about this paragraph. I would prefer it to be more straight-forward. What I want a reader to find out is:
* that a project is mandatory, because it provides fundamental functionality for services within the project to work properly
* you can have one or more projects
* that the total price of each project consists of the project's core package price + cost of resources of the services within the project (+ extra costs)
* how much each project core package actually costs 

Maybe you could move the third point above the ##Projects heading? Just an idea. What do you think?

Reply:
we good with this one?
<img width="990" alt="image" src="https://github.com/user-attachments/assets/911e826b-c6c0-4404-86bf-aad71031b275">

Reply:
yes :+1:

Reply:
cool then! you can merge the PR whenever you can, i'll resolve the conversations